src/components/Person/Person.jsx

```diff
@@ -1 +1,19 @@
-// export const Person = ({ person }) => ();
+import './Person.scss';
+
+export const Person = ({ person }) => {
+  const { name, age, isMarried, partnerName, sex } = person;
```

Comment:
If y have 3 or more properties - move to a new line
```suggestion
  const { 
    name, 
    age, 
    isMarried, 
    partnerName,
    sex
  } = person;
```